core/lib/libtomcrypt/src/misc/mem_neq.c

```diff
@@ -74,6 +74,11 @@ int mem_neq(const void *a, const void *b, size_t len)
       ++pb;
    }
 
+   ret |= ret >> 4;
+   ret |= ret >> 2;
+   ret |= ret >> 1;
+   ret &= 1;
+
```

Comment:
Sure, no problem. Let's hope the compiler never gets smart enough to see that it's the same thing or we'll be back at square one. :-)